app/src/main/java/org/fossasia/susi/ai/activities/ForgotPasswordActivity.java

```diff
@@ -95,7 +96,7 @@ public void signUp() {
         if(personalServer.isChecked()) {
             if(!CredentialHelper.checkIfEmpty(url,this) && CredentialHelper.isURLValid(url,this)) {
                 if (CredentialHelper.getValidURL(url,this) != null) {
-                    PrefManager.putBoolean("is_susi_server_selected", false);
+                    PrefManager.putBoolean(Constant.SUSI_SERVER, false);
                     PrefManager.putString("custom_server", CredentialHelper.getValidURL(url, this));
```

Comment:
Similarly use `public static final String CUSTOM_SERVER="custom_server";` in Constant.java and use it here.